canonical/snapcraft.io
# Convert brand store App component to TypeScript

## Done
Converted the brand store `App` component to TypeScript

## How to QA
- Go to https://snapcraft-io-4308.demos.haus/admin
- Check that the app renders without errors
